client/hostinfo_unix.cpp

```diff
@@ -1567,9 +1567,19 @@ inline long device_idle_time(const char *device) {
 static const struct dir_tty_dev {
     const char *dir;
     const char *dev;
+    const vector<string> ignore_list;
+
+    bool should_ignore(const std::string &devname) const {
```

Comment:
Sorry, my bad: `std::` prefix could be removed here since we're defining `using std::string;` above